cmake/common/sanitizers.cmake

```diff
@@ -36,6 +36,16 @@ option(SANITIZE_MEMORY "Enable MemorySanitizer" Off)
 option(SANITIZE_THREAD "Enable ThreadSanitizer" Off)
 option(SANITIZE_UNDEFINED "Enable UndefinedBehaviorSanitizer" Off)
 
+if(DEFINED ENV{PX4_ASAN})
+	set(SANITIZE_ADDRESS ON)
+elseif(DEFINED ENV{PX4_MSAN})
```

Comment:
Can you add these to the dev-guide? PX4_ASAN is already there: https://github.com/PX4/Devguide/blob/master/en/debug/simulation_debugging.md

Reply:
The others don't quite work yet, but I plan to get them working and added to builds.